definitions/fabrikate-eck/README.md

```diff
@@ -0,0 +1,43 @@
+# Elastic Cloud on Kubernetes - (ECK)
+
+This [Fabrikate](https://github.com/microsoft/fabrikate) definition generates the Kubernetes resource manifests for [ECK-Elastic Cloud on Kubernetes](https://github.com/elastic/cloud-on-k8s). ECK (Elastic Cloud on Kubernetes) encompasses the Elasticsearch operator.
+
+## Usage
+
+Add the following to your component.yaml:
```

Comment:
@NathanielRose why are we asking folks to modify the component.yaml as opposed to providing it to them?

Reply:
Removed and modified to instruct on how to customize sample cluster.